hazelcast/src/main/java/com/hazelcast/util/PhoneHome.java

```diff
@@ -138,52 +144,36 @@ public String convertToLetter(int size) {
         return letter;
     }
 
-    public Map<String, String> phoneHome(Node hazelcastNode, String version, boolean isEnterprise) {
-        String downloadId = "source";
-        InputStream is = null;
-        try {
-            is = getClass().getClassLoader().getResourceAsStream("hazelcast-download.properties");
-            if (is != null) {
-                final Properties properties = new Properties();
-                properties.load(is);
-                downloadId = properties.getProperty("hazelcastDownloadId");
-            }
-        } catch (IOException ignored) {
-            ignore(ignored);
-        } finally {
-            closeResource(is);
-        }
+    public Map<String, String> phoneHome(Node hazelcastNode) {
+        PhoneHomeParameterCreator parameterCreator = createParameters(hazelcastNode);
 
-        // calculate native memory usage from native memory config
-        NativeMemoryConfig memoryConfig = hazelcastNode.getConfig().getNativeMemoryConfig();
-        ClusterServiceImpl clusterService = hazelcastNode.getClusterService();
-        long totalNativeMemorySize = clusterService.getSize(DATA_MEMBER_SELECTOR) * memoryConfig.getSize().bytes();
-        String nativeMemoryParameter = (isEnterprise) ? Long.toString(MemoryUnit.BYTES.toGigaBytes(totalNativeMemorySize)) : "0";
+        String urlStr = BASE_PHONE_HOME_URL + parameterCreator.build();
+        fetchWebService(urlStr);
+
+        return parameterCreator.getParameters();
+    }
 
+    public PhoneHomeParameterCreator createParameters(Node hazelcastNode) {
+        ClusterServiceImpl clusterService = hazelcastNode.getClusterService();
         int clusterSize = clusterService.getMembers().size();
         Long clusterUpTime = clusterService.getClusterClock().getClusterUpTime();
         RuntimeMXBean runtimeMxBean = ManagementFactory.getRuntimeMXBean();
-        String licenseKey = hazelcastNode.getConfig().getLicenseKey();
         JetBuildInfo jetBuildInfo = hazelcastNode.getBuildInfo().getJetBuildInfo();
 
         PhoneHomeParameterCreator parameterCreator = new PhoneHomeParameterCreator()
-                .addParam("version", version)
+                .addParam("version", buildInfo.getVersion())
                 .addParam("m", hazelcastNode.getThisUuid())
-                .addParam("e", Boolean.toString(isEnterprise))
```

Comment:
Shouldn't we send `"false"` for `e` (instead of not sending anything for `e`)? This would preserve the old behavior and make any queries for `enterprise` property simpler in the future. You know better though, if it's common to have missing columns in the queries, we might leave this as is.